modules/ROOT/pages/cryptography-jce.adoc

```diff
@@ -7,53 +7,99 @@ The JCE strategy enables you to use the wider range of cryptography capabilities
 
 You can use cryptography capabilities in two ways:
 
-* PBE (Password-based encryption): Encrypt/sign content by providing only an encryption password.
-* Key-based Encryption: Similar to how PGP and XML encryption works, this lets you configure a symmetric or asymmetric key to perform encryption/signing operations.
+* Password-based encryption (PBE): Encrypt/sign content by providing only an encryption password.
+* Key-based Encryption: Similar to how PGP and XML encryption works, this method enables you to configure a symmetric or asymmetric key to perform encryption/signing operations.
 
 You can encrypt all, or part of a message using any of these two methods.
 
 == PBE
 
 This method applies a hash function over the provided password to generate a symmetric key that is compatible with standard encryption algorithms. Because PBE only requires a password, a global configuration element is not needed for the PBE operations.
 
-.Example: PBE Encryption
+=== Configure Password-Based Encryption From Anypoint Studio
+
+To configure PBE from Anypoint Studio, follow these steps:
+
+. From the Mule palette, add the Crypto module to your project.
++
+See xref:cryptography.adoc#install-crypto-module[Install the Extension] for instructions.
+. Select the desired operation, and drag the component to the flow:
++
+image::mruntime-crypto-pbe-add.png[crypto-pbe-add]
+. In the component view, configure the *Algorithm* and *Password* properties:
++
+image::mruntime-crypto-pbe-config.png[crypto-pbe-config]
+
+=== XML Examples
+
+The following are XML examples for each each of the PBE operations:
+
+* Example: PBE Encryption
```

Comment:
I wonder if we really need "Example:" in the subtitles of an Examples container topic?
I suggest omitting.